### PR TITLE
Changes for comment report settings

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
Issue #: https://github.com/bcgov/met-public/issues/

*Description of changes:*
- Added logic on comment model to exclude comments that are response to questions not to be displayed on public report
- Added new icons to represent the comment displayed/not displayed on comment review and read all comment pages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/comments/admin/textListing/index.tsx [hunk 1]
```diff
@@ -122,9 +124,40 @@ const CommentTextListing = () => {
                     {row.comments?.map((comment, index) => {
                         return (
                             <Grid key={index} item xs={12}>
-                                <Grid xs={12} item>
-                                    <MetLabel>{comment.label ?? 'Label not available.'} </MetLabel>
-                                    <MetParagraph>{' ' + comment.text}</MetParagraph>
+                                <Grid container direction="row" alignItems={'flex-start'} justifyContent="flex-start">
+                                    <Grid item xs={1} paddingTop={1}>
+                                        {comment.is_displayed ? (
```
Comment (djnunez-aot) [inline met-web/src/components/comments/admin/textListing/index.tsx:129]: Use When or If Then components from react-if

File: met-web/src/components/comments/admin/review/CommentReview.tsx [hunk 2]
```diff
@@ -270,11 +273,44 @@ const CommentReview = () => {
...
                             <Grid key={comment.id} item xs={12}>
                                 <Divider />
-                                <Grid xs={12} item paddingTop={2}>
-                                    <MetLabel>{comment.label ?? 'Label not available.'}</MetLabel>
-                                </Grid>
-                                <Grid xs={12} item>
-                                    <MetParagraph>{comment.text}</MetParagraph>
+                                <Grid container direction="row" alignItems={'flex-start'} justifyContent="flex-start">
+                                    <Grid item xs={1} paddingTop={3}>
+                                        {comment.is_displayed ? (
```
Comment (djnunez-aot) [inline met-web/src/components/comments/admin/review/CommentReview.tsx:278]: Use react-if components

**Review by djnunez-aot — COMMENTED** [review]
Everything looks good just add react-if components instead of ternary operators

File: met-web/src/components/comments/admin/textListing/index.tsx (hunk 1 above)
Comment (VineetBala-AOT) [reply met-web/src/components/comments/admin/textListing/index.tsx]: updated, thanks.

File: met-web/src/components/comments/admin/review/CommentReview.tsx (hunk 2 above)
Comment (VineetBala-AOT) [reply met-web/src/components/comments/admin/review/CommentReview.tsx]: updated

**Review by djnunez-aot — APPROVED** [review]

**Pull request merged**